5-app-infra/README.md

```diff
@@ -75,14 +76,14 @@ This Compute Engine instance is created using the base network from step `3-netw
 
 ### Deploying with Cloud Build
 
-1. Get the project ID from the previous procedure. Run the following in the
-   `4-projects/business_unit_1/shared` folder.
+1. Get the Infra Pipeline project ID from the previous step. Run the following in the
+   `gcp-projects/business_unit_1/shared` folder.
```

Comment:
This makes the assumption you are running terraform locally/have it setup.

Reply:
Yes, because step in step 4-projects we tell the user that they have to run manually 

- gcp-projects/business_unit_1/shared
- gcp-projects/business_unit_2/shared

Because development, non-production, and production depends on the result of the shared folder, the same dependency with have in step 3-networks regarding the DNS hub